src/types/clusters_mgmt.v1/index.ts

```diff
@@ -15874,6 +15891,8 @@ export interface components {
       category?: components['schemas']['MachineTypeCategory'];
       /** @description Link to the cloud provider that the machine type belongs to. */
       cloud_provider?: components['schemas']['CloudProvider'];
+      /** @description Features available to the particular MachineType */
+      features?: components['schemas']['MachineTypeFeatures'];
```

Comment:
New feature property for machine types, introduced to support the `win_li` feature

cc @LiorKGOW you were waiting for this

Reply:
Definitely! 
This PR also includes the 2nd change related to Windows LI for Node Pools (ImageType) 🤩